resources/builder.inc.sh

```diff
@@ -441,31 +441,47 @@ _builder_cleanup_deps() {
 # Child scripts
 #------------------------------------------------------------------------------------------
 
+#
+# Starts a child script build, passing current build dependency, inheritable
+# options, and timing status through as parameters. This should be used rather
+# than calling the build script directly, to avoid multiple builds of
+# dependencies.
+#
+# ### Parameters
+#
+# * 1: `script`      path to script, relative to root of repo
+# * 2: `action`      action(s)+target(s) for the child script to run, comma separated, at least one
+#
```

Comment:
Is there a preference for including the target(s) in the 2nd param vs having an optional 3rd param like before?

The header to `_builder_execute_child_script` mentions the optional 3rd param.

Reply:
I wanted to match the command line of `target:action` rather than having `target :action` looks quite different.

But I realised while responding to your question that we need to be able to support options as well as targets/actions, so reworking this now.